src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderAlgorithmPicked.cpp

```diff
@@ -401,9 +394,9 @@ namespace OpenMS
         isotope_distributions_[index].trimmed_left = size_before - d.size();
         d.trimRight(intensity_percentage_optional_);
 
-        for (IsotopeDistribution::Iterator it = d.begin(); it != d.end(); ++it)
+        for (auto & it : d)
```

Comment:
dont name it `it` since its not an iterator. Use a name corresponding to the type.

Reply:
Renamed to peak